### ISSUE TITLE
Input string was not in a correct format

### ISSUE THREAD

**Issue by Soderhamn** [Enumer8/Anim8orTransl8or#3]
Hi, first of all I want to say that i like the idea with Anim8orTransl8or. This project probably makes it possible to export Anim8or models into the Godot game engine (which supports Collada (.dae)-files). At least that is what I am trying to find out.

However, I have got a problem. When I run Anim8orTransl8or.Gui.exe i get the error message "Input string was not in a correct format". When I run Anim8orTransl8or.Cli.exe I get the following error message: 

`Unhandled exception. System.FormatException: Input string was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
   at Anim8orTransl8or.An8.Serializer.ParseFloat(StreamReaderEx sr)
   at Anim8orTransl8or.An8.Serializer.ParseChunk(StreamReaderEx sr, Type t)
   at Anim8orTransl8or.An8.Serializer.ParseChunk(StreamReaderEx sr, Type t)
   at Anim8orTransl8or.An8.Serializer.ParseChunk(StreamReaderEx sr, Type t)
   at Anim8orTransl8or.An8.Serializer.Deserialize(Stream s)
   at Anim8orTransl8or.Cli.Program.Main(String[] args)`

I've tried both v.0.7.1 and v. 0.7.2 on my own an8-file and on the example Cat.an8-file with the same result.

OS: Windows 11. .NET version information: 

> .NET SDK:
 Version:   7.0.102
 Commit:    4bbdd14480

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22631
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\7.0.102\

Host:
  Version:      7.0.2
  Architecture: x64
  Commit:       d037e070eb

.NET SDKs installed:
  7.0.102 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.31 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 6.0.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

**Comment by CounterMatter** [comment]
I should have thought about internationalization. I'll make a new release.

**Comment by Enumer8** [comment]
Sorry for overlooking internationalization issues! I made a new release which should resolve the issue: https://github.com/Enumer8/Anim8orTransl8or/releases/tag/v0.7.3

Let me know how it works for you. I haven't tried importing any files into Godot, so also let me know how that goes. Good luck with your project.

**Comment by Soderhamn** [comment]
The issue is resolved, I can now convert .an8-files to Collada, thank you! 

I don't have that much experience working with 3d in Godot, so I may have missed something, but the generated Collada files does not seem to work in Godot. I tried with both version 3.5.3 and 4.2 of the engine. I get the error message: "Attribute not found: meter."

A workaround I found was to simply import the Collada files into Blender and re-export the model into .gltf-format (which is the recommended file format for importing 3d assets into Godot anyway, (.glb didn't work)). By doing that I have an animated 3d-object created in Anim8or inside the Godot engine. Pretty cool, since it was previously only possible to export static objects.

**Comment by Enumer8** [comment]
Thanks for the update. Glad to hear it is working better now.

I was curious, so I tried Godot. I didn't get an errors, but it also didn't really look right either. I tried other animated dae files I found online, and it seemed like none of them worked very well either.

Looking back, I probably should have just implemented gltf format the first time around instead of dae. I thought dae would be a universal and open format, but in reality it seems like most programs just have very bad/incomplete support. I am planning on supporting gltf directly one day, but I don't know when.

Thanks again.
